docs-core/src/main/resources/db/update/dbupdate-028-0.sql

```diff
@@ -0,0 +1,3 @@
+alter table T_TAG alter column TAG_NAME_C type varchar(50);
+alter table T_TAG alter column TAG_NAME_C set not null;
```

Comment:
The change to `varchar(50)` and adding the `not null` constraint could have been done in 1 line but since the migration is already applied to the server, it has to stay as it is :)